lib/whitehall_adapter.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class WhitehallAdapter
+  def self.get_document_list(args = {})
```

Comment:
I'd imagine it'd be more consistent with GDS API Adapters and Ruby idioms to drop the `get_` from the endpoint methods.

Reply:
Done.  Renamed to `document_list` and `document_export.